### PR TITLE
More Data Prepper changes

### PR DESCRIPTION
**Pull request by hdhalter**
### Description
I tripped over myself when merging the previous Data Prepper changes, and had to re-add the {copy} commands. I made a few more tweaks while I was in there. 

### Issues Resolved
N/A, just adds to a previous PR: https://github.com/opensearch-project/documentation-website/pull/3284

### Checklist
- [x] By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license and subject to the [Developers Certificate of Origin](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Review by carolxob — APPROVED** [review]
LGTM.

File: _data-prepper/getting-started.md [hunk 1]
```diff
@@ -93,24 +96,27 @@ For Data Prepper 2.0 or later, use this command:
...
 
 ```
 docker run --name data-prepper -p 4900:4900 -v ${PWD}/pipelines.yaml:/usr/share/data-prepper/pipelines.yaml -v ${PWD}/data-prepper-config.yaml:/usr/share/data-prepper/data-prepper-config.yaml opensearchproject/data-prepper:1.x
 ```
+{% include copy.html %}
 
 Once Data Prepper is running, it processes data until it is shut down. Once you are done, shut it down with the following command:
 
 ```
 curl -X POST http://localhost:4900/shutdown
```
Comment (ariamarble) [inline _data-prepper/getting-started.md:111]:
```
POST /shutdown
```

File: _data-prepper/getting-started.md [hunk 2]
```diff
@@ -93,24 +96,27 @@ For Data Prepper 2.0 or later, use this command:
...
 ```
+{% include copy.html %}
```
Comment (ariamarble) [inline _data-prepper/getting-started.md:113]:
```
{% include copy-curl.html %}
```

**Review by ariamarble — APPROVED** [review]
looks good, but I update the API call and copy command. thanks!

**Pull request merged**